tasks.py

```diff
@@ -365,6 +367,25 @@ def deploy_tegan(c: Any) -> None:
     )
     deploy_nixos([host])
 
+@task
+def deploy_ace(c: Any) -> None:
+    """
+    Deploy to morello server
+    """
+    host = DeployHost(
+        "yasmin.dos.cit.tum.de",
+        user="root",
+        forward_agent=True,
+        command_prefix="ace",
```

Comment:
Is ace not strong enough to build itself? We could also make yasmin a remote builder of ace otherwise. This makes updating a bit more stream-lined.

Reply:
No, ace is very slow. Building takes ~20 mins on yasmin and >2 hrs on ace.